### PR TITLE
PK Mod: Misc Spawn Lists, tweaks to critters

### PR DESCRIPTION
**Pull request by pisskop**
Updates of small tweaky nature, plus the addtion of the more complete spawn list.

### CODE REVIEW

File: data/mods/PKs_Rebalance/pk_critter_triffid.json [hunk 1]
```diff
@@ -253,11 +253,11 @@
     "hp":150,
     "death_function":"NORMAL",
     "description":"An animated mass of roots and vines, creeping along the ground with alarming speed.  The tangle is thick enough that the center from which they grow is concealed.  This one looks tough.",
-    "flags":["SEES", "HEARS", "GOODHEARING", "PUSH_MON", "NOHEAD", "HARDTOSHOOT", "GRABS", "SWIMS", "PLASTIC"]
+    "flags":["SEES", "HEARS", "GOODHEARING", "PUSH_MON", "NOHEAD", "HARDTOSHOOT", "GRABS", "PLASTIC"]
     },{
     "type" : "MONSTER",
     "id" : "mon_vinebeast_pk",
-    "name": "tangela",
+    "name": "tangala",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_critter_triffid.json]: dont revert to your typo version...

**Comment by pisskop** [comment]
Its not a typo, its intentionally named such

File: data/mods/PKs_Rebalance/pk_critter_triffid.json [hunk 2]
```diff
@@ -493,8 +493,8 @@
             "min_mul" : 0.80, "//" : "dionaea bitch"
         },
         ["VINE", 15], ["RANGED_PULL", 3]],
-        "description":"A rotund armored 'head' with a massive maw dominates this monster's form.  Its head hangs low when its not in use, saving energy for its coordinated slither along the ground.",
-        "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "VENOM", "SWIMS", "BASHES", "GROUP_BASH", "PUSH_MON", "NOHEAD"],
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_critter_triffid.json]: Are you going to say this is also intentional?

File: data/mods/PKs_Rebalance/pk_critter_triffid.json [hunk 3]
```diff
@@ -493,8 +493,8 @@
...
+        "description":"A rotund armored 'head' with a massive maw dominates this monster's form.  Its head hangs low when its not in use, saving enegery for its coordinated slither along the ground.",
+        "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "VENOM", "BASHES", "GROUP_BASH", "PUSH_MON", "NOHEAD"],
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_critter_triffid.json]: Are you going to say this is also intentional?

File: data/mods/PKs_Rebalance/pk_critter_triffid.json [hunk 4]
```diff
@@ -493,8 +493,8 @@
             "min_mul" : 0.80, "//" : "dionaea bitch"
         },
         ["VINE", 15], ["RANGED_PULL", 3]],
-        "description":"A rotund armored 'head' with a massive maw dominates this monster's form.  Its head hangs low when its not in use, saving energy for its coordinated slither along the ground.",
-        "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "VENOM", "SWIMS", "BASHES", "GROUP_BASH", "PUSH_MON", "NOHEAD"],
+        "description":"A rotund armored 'head' with a massive maw dominates this monster's form.  Its head hangs low when its not in use, saving enegery for its coordinated slither along the ground.",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_critter_triffid.json]: Are you going to say this is also intentional?

**Comment by cainiaowu** [comment]
Using unconventional name will only confuse non-native english speaking translator.

**Comment by pisskop** [comment]
sure it is.

you gonna say i fixed half the typos bit kept the others on purpose after ive demonstrated my amateurish style  :v:

The point of the -gala suffix thing was to have it be a blatant knockoff.  id prefer to keep that subtle humour in but its really not worth fighting over

**Pull request merged**